
# msg to set PID current point

uint8 dof

float32 current
